src/components/elements/ImageGallery/ImageGallery.tsx

```diff
@@ -13,6 +13,9 @@ import { VARIANT_PAGINATION_TEXT_16 } from "@/components/extensive/Pagination/Pa
 import { useModalContext } from "@/context/modal.provider";
 import { Option } from "@/types/common";
 
+import Button from "../Button/Button";
+import FilterSearchBox from "../TableFilters/Inputs/FilterSearchBox";
+import Toggle from "../Toggle/Toggle";
```

Comment:
please change this imports to not use relative paths `../Button...` should be `@/...`

Reply:
done